packages/stencil/components/input/src/components/osds-input/osds-input.e2e.ts

```diff
@@ -223,18 +298,53 @@ describe('e2e:osds-input', () => {
 
   describe('method:setFocus', () => {
 
-    it('should set setFocus', async () => {
+    it('should be focusable', async () => {
       await setup({ attributes: { type: OdsInputType.number } });
       await page.waitForChanges();
-      let value = el.getAttribute('hasFocus');
-      expect(value).toBeNull();
+      
+      await el.callMethod('setFocus');
+      await page.waitForChanges();
+  
+      const isFocused = await page.evaluate(() => {
+        const element = document.querySelector('osds-input');
+        return document.activeElement === element;
+      });
+      expect(isFocused).toBe(true);
+    });
 
+    it('should be focusable with tab', async () => {
+      await setup({ attributes: { type: OdsInputType.number } });
+      await page.waitForChanges();
+    
+      // First, we set the focus to another element
+      await page.focus('#anotherInput');
+    
+      // Then we simulate pressing 'Tab' to move focus to the next focusable element
+      await page.keyboard.press('Tab');
+    
+      // We can now check if the input is focused
+      const isFocused = await page.evaluate(() => {
+        const element = document.querySelector('osds-input');
+        return document.activeElement === element;
+      });
+    
+      expect(isFocused).toBe(true);
+    });    
+  
+    it('should not be focusable when disabled', async () => {
+      await setup({ attributes: { type: OdsInputType.number, disabled: true } });
+      await page.waitForChanges();
+  
       await el.callMethod('setFocus');
       await page.waitForChanges();
-      value = el.getAttribute('hasFocus');
-      expect(value).toEqual('');
+  
+      const isFocused = await page.evaluate(() => {
+        const element = document.querySelector('osds-input');
+        return document.activeElement === element;
+      });
+      expect(isFocused).toBe(false);
     });
-  });
+  });  
```

Comment:
@Leotheluck setup your IDE so that trailing spaces are automatically removed on save, otherwise all your PRs will be filled with useless changes like this